src/storage/src/source/reclock.rs

```diff
@@ -313,7 +313,9 @@ impl ReclockOperator {
             read_handle,
             listener,
             now,
-            update_interval_ms: Timestamp::try_from(update_interval.as_millis())
+            update_interval_ms: update_interval
+                .as_millis()
+                .try_into()
```

Comment:
Nit, but readability is reduced here (if you wanted to know what the type is). Is something bad about the previous formulation?

Reply:
`update_interval_ms` is (unexpectedly) a u64, not a Timestamp, so we can't use the previous code.